app/views/play/level/LevelGoalsView.coffee

```diff
@@ -37,17 +39,15 @@ module.exports = class LevelGoalsView extends CocoView
   constructor: (options) ->
     super options
     @level = options.level
-    
-  afterRender: ->
-    @levelGoalsComponent = new LevelGoals({
-      el: @$('.goals-component')[0],
-      store
-      propsData: { showStatus: true }
-    })
+
+    @setState({ showStatus: true })
```

Comment:
Relalted to my costructor comment above, we could set the initial state using the constructor, and then use `@setState` for changing it afterwards